packages/avro-kafkajs/src/SchemaRegistry.ts

```diff
@@ -91,6 +97,12 @@ export class SchemaRegistry {
     return await getSubjectVersionSchema(this.uri, subject, version);
   }
 
+  public async getSubjectLastVersionSchema(subject: string): Promise<Schema> {
+    const versions = await getSubjectVersions(this.uri, subject);
```

Comment:
We came across a few times where the schema registry would not return an array but a null values instead, you may want to check that this is actually an array